app/forms/bulk_submission_form.rb

```diff
@@ -25,6 +28,7 @@ def self.max_file_size
   def save
     validate
     return false unless valid?
+    return false unless file_contents_valid? # only attempt to validate the file contents if initial validations pass
```

Comment:
we could a rails style validator such as  

```
# contrived example
validate  :file_content, if: { file_type.valid? } 
```


https://api.rubyonrails.org/v5.1/classes/ActiveModel/Validations/ClassMethods.html#method-i-validates